### DISCUSSION TITLE
Getting error as: Exception: ('Error adding DDL', {'error': {'code': -32600, 'message': 'User yogeswariyrsk@gmail.com not in model employee_db'}, 'jsonrpc': '2.0'})

### DISCUSSION THREAD

**Discussion by yogeswari19** [vanna-ai/vanna#676] [Q&A]
I tried using Vanna's API and connected the model to a DB in my postgres. But I got the following error:
![Screenshot (25)](https://github.com/user-attachments/assets/e13f77e8-cea6-4353-a70a-9dd092e05e5c)
![Screenshot (26)](https://github.com/user-attachments/assets/8196f50c-bc70-45cc-bcde-c063ef18585e)

**Comment by tuliofariasoncase** [comment]
same problem here, did anyone find any possible solution to this?
